package.json

```diff
@@ -110,6 +111,7 @@
     "react-redux": "^5.0.6",
     "react-router": "3.2.0",
     "react-select": "^1.0.0-rc.10",
+    "react-select-plus": "^1.2.0",
```

Comment:
This should also include a change to `yarn.lock`.

If you run `yarn` locally, it should add the exact versions of these dependencies to the lock file.